test/lit/passes/gufa.wast

```diff
@@ -1,22 +1,22 @@
 ;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
-;; RUN: foreach %s %t wasm-opt -all --gufa -S -o - | filecheck %s
+;; RUN: foreach %s %t wasm-opt -all --hybrid --gufa -S -o - | filecheck %s
```

Comment:
We have `gufa-refs.wast` for GC tests - please move the new tests to there, and then this file won't need `--hybrid`.